packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -5,13 +5,14 @@
 
 Name: rubygem-%{gem_name}
 Version: 7.0.0
-Release: 1%{?foremandist}%{?dist}
+Release: 2%{?foremandist}%{?dist}
 Summary: Foreman plug-in for displaying OpenSCAP audit reports
 License: GPLv3
 URL: https://github.com/theforeman/foreman_openscap
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
+Requires: scap-security-guide
```

Comment:
Note the line above `# start specfile generated dependencies` this is to indicate that this section is auto-generated. So our solution for that is to put additional requires above that line:

```suggestion
Requires: scap-security-guide
# start specfile generated dependencies
```